ast/parser.go

```diff
@@ -2631,7 +2631,7 @@ func (p *parser) read() {
 	}
 
 	if rn == utf8.RuneError {
-		if n == 1 {
+		if n > 0 {
```

Comment:
This indicates your installed version of `pigeon` is out-of-date. Please `rm $GOPATH/bin/pigeon` and `make deps` to re-install `pigeon`.

This needs to be removed from the PR. Let me know if you want me to do this.